src/components/MergeInfographic.tsx

```diff
@@ -111,20 +84,55 @@ const MergeInfographic: React.FC<IProps> = ({ className }) => {
     translateMessageId(id, intl)
 
   return (
-    <Container
+    <VStack
       className={className}
       role="img"
       aria-label={translateMessageId(
         "page-upgrades-merge-infographic-alt-text",
         intl
       )}
+      position="relative"
+      width="100%"
+      sx={{
+        isolation: "isolate",
+        aspectRatio: `${25 / 11}`,
+      }}
     >
```

Comment:
Because the `aspectRatio` prop is used, I think it would be better to use the [AspectRatio](https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/layout/src/aspect-ratio.tsx) component instead, else `Box`. 

`VStack` doesn't make sense here because there are no visually stacking elements in the DOM flow (just about everything is absolute-positioned instead), and `VStack` by default renders a `marginTop` for each child element. In this case, it's simply rendering unnecessary space. :)

With `AspectRatio` you can use the `ratio` prop instead with the needed value.

**NOTE**: All the child components will need to be wrapped in a `div` because `AspectRatio` is looking for a single child. Else it'll break the SSR.

```suggestion
<AspectRatio
    className={className}
    role="img"
    aria-label={translateMessageId(
      "page-upgrades-merge-infographic-alt-text",
       intl
    )}
    position="relative"
    width="100%"
    ratio={25 / 11}
    sx={{
      isolation: "isolate",
    }}
>
```

This component will provide more robust positioning and sizing logic. I wonder if @pettinarip agrees on this approach instead of just using `Box`! 😄

Reply:
Great suggestion, yes. Lets try to use as much as we can the Chakra components. And good point about the usage of `VStack`, I think its not necessary here.